ManatiCR/artavia
# AR-40 Que hacemos #3

[AR-40](https://manati.atlassian.net/browse/AR-40)
[Figma designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
  This PR depends of this one: [AR-55](https://github.com/ManatiCR/artavia/pull/8)
---
- Create the two hero blocks in the page que hacemos
- Create an entityque to select which clientes to show in the page que hacemos 
**Steps to test**
- Run `./scripts/artavia_local_install.sh` (New default content was added so it has to be created)
- Create clientes nodes
- Add some clientes to the entityque http://artavia.docker/es/admin/structure/entityqueue/clientes_destacados/clientes_destacados (here you select which clientes to show in the page que hacemos
- Go to the page http://artavia.docker/es/que-hacemos 
- There should be two blocks with sample text, see designs (The image of the first block will be added with styles as a background)
- The section enfoques de trabajo should show the following taxonomy terms: Estrategia, Sostenibilidad, Innovación, Responsabilidad Social
- The section Algunos de nuestros clientes, should show the clientes selected in the entityque clientes destacados
